lib/index.js

```diff
@@ -90,15 +90,32 @@ Promise.prototype.then = function(success, error){
 };
 
 
+
 // Attributes that remain on the root of the json payload for a recipient
 var	rootAttributes = {
 	email: 'email',
 	headers: 'headers',
 	layoutId: 'layout_id',
 	name: 'name',
-	replyTo: 'reply_to'
+	replyTo: 'reply_to',
+	attachments: 'attachments'
 };
 
+// Add attachment
+var attachments = []
+var add_attachments = function(name, content){
+	/**
+	* Adds a single attachment to the list.
+	*
+	* @param {string} name, name of the file with or without extension
+	* @param {string} content , content of the file
+	*
+	* @return list of attachments
+	*/
+	attachments.push({'name':name,'data':window.btoa(content)})
```

Comment:
The apostle.js library is used in both browser and server environments. Node doesn't provide `window.btoa`. Please provide an alternative for window.btoa, e.g. 
```
var btoa; 
if (undefined === window || undefined === window.btoa) {
    btoa = require('btoa');
} else {
    btoa = window.btoa;
}
```

Reply:
Sorry missed that. Will do the changes.